### PR TITLE
Fix: ciphersuites length equals to 1(expecting 0) when using the default value.

### PR DESCRIPTION
**Pull request by xuezhaojun**
Fix: https://github.com/kubernetes-sigs/apiserver-network-proxy/issues/503

### CODE REVIEW

**Comment by jkh52** [comment]
/ok-to-test

Thanks for fixing!

File: cmd/server/app/options/options.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func (o *ProxyRunOptions) Flags() *pflag.FlagSet {
 	flags.IntVar(&o.KubeconfigBurst, "kubeconfig-burst", o.KubeconfigBurst, "Maximum client burst (proxy server uses this client to authenticate agent tokens).")
 	flags.StringVar(&o.AuthenticationAudience, "authentication-audience", o.AuthenticationAudience, "Expected agent's token authentication audience (used with agent-namespace, agent-service-account, kubeconfig).")
 	flags.StringVar(&o.ProxyStrategies, "proxy-strategies", o.ProxyStrategies, "The list of proxy strategies used by the server to pick a backend/tunnel, available strategies are: default, destHost.")
-	flags.StringVar(&o.CipherSuites, "cipher-suites", o.CipherSuites, "The comma separated list of allowed cipher suites. Has no effect on TLS1.3. Empty means allow default list.")
+	flags.StringArrayVar(&o.CipherSuites, "cipher-suites", o.CipherSuites, "The comma separated list of allowed cipher suites. Has no effect on TLS1.3. Empty means allow default list.")
```
[the remark this answers is not in the document]
Comment (xuezhaojun) [reply cmd/server/app/options/options.go]: Thanks for the review. The code has been updated and a test case added.

**Comment by jkh52** [comment]
/lgtm

I prefer to wait a bit of time for additional eyes before approving.

/assign @Tamas-Biro1 

Does it look good to you, Tamás?

**Review by Tamas-Biro1 — APPROVED** [review]
looks good to me, thanks for fixing it.

**Pull request merged**